src/Debugger-Oups/OupsDebugRequest.class.st

```diff
@@ -55,8 +55,15 @@ OupsDebugRequest class >> newForContext: aContext [
 
 { #category : #'instance creation' }
 OupsDebugRequest class >> newForException: anException [
```

Comment:
Please sync this changes with actual repository if it exists. I did not find a separate project for Debugger-Oops.